test/unit/resource.ts

```diff
@@ -99,6 +99,23 @@ describe('Resource', () => {
 
   });
 
+  describe('Resource.ActiveRefreshes', () => {
+
+    it('duplicated headers key are handled', async () => {
+
+      const res: any = getFakeResource('https://example.org/return-request');
+
+      const headers = new Headers({'foo': 'bar'});
+      headers.append('foo', 'baz');
+      const headers2 = new Headers({'foo': 'bar'});
+
+      res.activeRefreshes.put(res.uri, {headers: headers}, null);
```

Comment:
`activeRefreshes` is private. Tests should not break encapsulation.

Reply:
Done